docs/docs-content/clusters/pcg/manage-pcg/pcg-upgrade.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_label: "Upgrade a PCG"
+title: "Upgrade a PCG"
+description: "Learn how to upgrade a Private Cloud Gateway (PCG) in Palette."
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["pcg"]
+---
+
+A PCG upgrade event can be broken down into two categories: Palette agent updates and cluster profile updates. Depending
```

Comment:
"cluster profile updates" > it's more a "hidden" cluster profile, specific to the PCG type (depending on the cloud). I see it's mentioned below in the page, but I think we should also mention a quick sentence at the top to avoid someone reading through more than 50% of the page to discover this concept; just my 2c.

Reply:
Maybe the page should use "PCG cluster profile" in all occurrences as well?